### PR TITLE
Use yaml for parse Facebook credentials + Lint + give all Chromedriver binaires

### PR DESCRIPTION
**Pull request by SkYNewZ**
- Lint Python files with [Black](https://pypi.org/project/black/)
- Lint `README.md`
- Provide `.gitignore` file from http://gitignore.io/api/python
- Update documentation to specify that Facebook accounts secured with 2FA will not work
- Give all Chromedriver version and dynamically use them

You should use [Git LFS](https://git-lfs.github.com) to track Chromedriver binaries and pictures

Fix https://github.com/harismuneer/Ultimate-Facebook-Scraper/issues/70

### CODE REVIEW

**Comment by harismuneer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/Ultimate-Facebook-Scraper/Ultimate-Facebook-Scraper/pullRequest?prid=4586331&bid=15400869)

File: src/scraper.py [hunk 1]
```diff
@@ -631,70 +789,78 @@ def login(email, password):
...
 # -----------------------------------------------------------------------------
 # -----------------------------------------------------------------------------
 
+
 def main():
-    with open('credentials.txt') as f:
-        email = f.readline().split('"')[1]
-        password = f.readline().split('"')[1]
+    with open("credentials.yaml", "r") as ymlfile:
+        cfg = yaml.load(stream=ymlfile, Loader=yaml.BaseLoader)
```
Comment (harismuneer) [inline src/scraper.py]: Codacy Issue found: [Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4586331)

**Comment by harismuneer** [comment]
@SkYNewZ thanks for reformating the files. I think I will have to create a linting file in the project itself so that everyone follows the same coding style. Also, I will surely transfer the binaries to Git LFS. 

Thank you for your contributions, highly appreciate that!

**Comment by SkYNewZ** [comment]
If you choose to use Black, there is not configurations file need. Black force usage of its rules 😀

No probleme, its a pleasure

**Pull request merged**